internal/gateway/getbgpneighbours.go

```diff
@@ -0,0 +1,49 @@
+package gateway
+
+import (
+	"context"
+	"fmt"
+	"strconv"
+
+	"github.com/spf13/cobra"
+)
+
+func (c *Client) GetBgpNeighbours() *cobra.Command {
+	var bgpNeighbourID string
+
+	// getGwayBgpDynamicNeighbourCmd represents command to get Metal Gateway Dynamic Neighbour by ID.
+	getGwayBgpDynamicNeighbourCmd := &cobra.Command{
+		Use:   `get-bgp-dynamic-neighbours`,
```

Comment:
Equinix docs/tools/etc. generally use American English spellings, so we should align with that throughout this PR.

```suggestion
		Use:   `get-bgp-dynamic-neighbors`,
```

Reply:
Corrected at all the references